evaluator/builtin_string.go

```diff
@@ -637,3 +640,102 @@ func builtinRpad(args []types.Datum, ctx context.Context) (d types.Datum, err er
 
 	return d, nil
 }
+
+// https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_bit-length
+func builtinBitLength(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
+	// args[0] string
+	if args[0].IsNull() {
+		return d, nil
+	}
+
+	str, err := args[0].ToString()
+	if err != nil {
+		return d, errors.Trace(err)
+	}
+
+	d.SetInt64(int64(len(str) * 8))
+
+	return d, nil
+}
+
+// https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_char
+func builtinChar(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
+	// args are number or string; last one is charset
```

Comment:
Make comment readable: The kinds of args are int or string, and the last one represents charset.